stf/src/cli.rs

```diff
@@ -158,8 +159,9 @@ pub fn cmd<'a>(
                         amount
                     );
                     let (mrenclave, shard) = get_identifiers(matches);
-                    let nonce = 0; // FIXME: hard coded for now
-                                   // generate trusted call signed
+                    let encoded_nonce = get_nonce(matches, &sr25519_core::Public::from(from.public()).into());
+                    let nonce = Index::decode(&mut encoded_nonce.as_slice()).unwrap();
```

Comment:
if `encoded_nonce` isn't used later, I'd prefer a one-liner for readability

Reply:
fixed